client/App.jsx

```diff
@@ -9,7 +9,7 @@ class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      'product_id': '22124'
```

Comment:
Did you mean to change the product_id at the app level? Doesn't really matter I guess...